modules/tripexplorer/SubwayTripGraphs.tsx

```diff
@@ -62,6 +64,20 @@ export const SubwayTripGraphs: React.FC<SubwayTripGraphsProps> = ({
               toStation={toStation}
             />
           </WidgetDiv>
+          <WidgetDiv>
+            <WidgetTitle
+              title="Speeds"
+              subtitle="Speed between stops"
```

Comment:
We should say 
```suggestion
              subtitle="Average speed between stops"
```
since there are several speed limits between a pair